inspire_mitmproxy/http.py

```diff
@@ -149,17 +155,21 @@ def to_mitmproxy(self) -> HTTPRequest:
             path=parsed_url.path,
             http_version=self.http_version,
             headers=self.headers.to_mitmproxy(),
-            content=self.body.encode(self.original_encoding),
+            content=self.body,
         )
 
     def to_dict(self) -> Dict[str, Any]:
         return {
             'method': self.method,
             'uri': self.url,
-            'body': self.body,
+            'body': self.body.decode(self.original_encoding),
             'headers': self.headers.to_dict(),
         }
 
+    @property
+    def str_body(self):
+        return self.body.decode(self.original_encoding)
```

Comment:
this is not used anywhere no?

Reply:
tests/integration/test_base_service.py @ 101, tests/unit/test_dispatcher.py @ 80, but tbf I can remove those usages

Reply:
Done :+1: